db/schema.rb

```diff
@@ -78,6 +78,33 @@
     t.index ["rank"], name: "index_faqs_on_rank"
   end
 
+  create_table "invoice_files", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
+    t.uuid "invoice_id"
+    t.uuid "uploaded_by_id"
+    t.boolean "archived", default: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["invoice_id"], name: "index_invoice_files_on_invoice_id"
+    t.index ["uploaded_by_id"], name: "index_invoice_files_on_uploaded_by_id"
+  end
+
+  create_table "invoices", id: :uuid, default: -> { "uuid_generate_v4()" }, force: :cascade do |t|
+    t.uuid "partner_id"
+    t.datetime "date"
+    t.string "amount", default: "0"
+    t.string "finalized_amount"
+    t.boolean "paid", default: false
+    t.uuid "paid_by_id"
+    t.datetime "payment_date"
```

Comment:
Ditto here for `t.date`

Reply:
This is it - https://github.com/brave-intl/publishers/pull/1620/commits/ab7050f639094d5c771697143338194f0046ad3c